{
	"vulinfoid": "JVNDB-2018-001955",
	"vulinfodata": {
		"title": "WavPack におけるバッファエラーの脆弱性",
		"vulinfodescription": {
			"overview": "WavPack には、バッファエラーの脆弱性が存在します。"
		},
		"affected": {
			"affecteditem": [
				{
					"name": "Wavpack project",
					"productname": "Wavpack",
					"cpe": {
						"text": "cpe:/a:wavpack_project:wavpack",
						"version": "2.2"
					},
					"versionnumber": [
						"5.1.0"
					]
				}
			]
		},
		"impact": {
			"cvss": [
				{
					"version": "2.0",
					"severity": {
						"text": "Medium",
						"type": "Base"
					},
					"base": "6.8",
					"vector": "AV:N/AC:M/Au:N/C:P/I:P/A:P"
				},
				{
					"version": "3.0",
					"severity": {
						"text": "High",
						"type": "Base"
					},
					"base": "7.8",
					"vector": "CVSS:3.0/AV:L/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H"
				}
			],
			"impactitem": {
				"description": "情報を取得される、情報を改ざんされる、およびサービス運用妨害 (DoS) 状態にされる可能性があります。"
			}
		},
		"solution": {
			"solutionitem": {
				"description": "ベンダより正式な対策が公開されています。ベンダ情報を参照して適切な対策を実施してください。"
			}
		},
		"related": {
			"relateditem": [
				{
					"type": "vendor",
					"name": "Debian Bug report logs",
					"vulinfoid": "889276",
					"url": "https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=889276"
				},
				{
					"type": "vendor",
					"name": "GitHub",
					"vulinfoid": "stack buffer overflow while running wavpack #27",
					"url": "https://github.com/dbry/WavPack/issues/27"
				},
				{
					"type": "vendor",
					"name": "GitHub",
					"vulinfoid": "issue #27, do not overwrite stack on corrupt RF64 file",
					"url": "https://github.com/dbry/WavPack/commit/d5bf76b5a88d044a1be1d5656698e3ba737167e5"
				},
				{
					"type": "advisory",
					"name": "Common Vulnerabilities and Exposures (CVE)",
					"vulinfoid": "CVE-2018-6767",
					"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-6767"
				},
				{
					"type": "advisory",
					"name": "National Vulnerability Database (NVD)",
					"vulinfoid": "CVE-2018-6767",
					"url": "https://nvd.nist.gov/vuln/detail/CVE-2018-6767"
				},
				{
					"type": "cwe",
					"name": "JVNDB",
					"vulinfoid": "CWE-119",
					"url": "https://jvndb.jvn.jp/ja/cwe/CWE-119.html",
					"title": "バッファエラー"
				}
			]
		},
		"history": {
			"historyitem": [
				{
					"historyno": "1",
					"datetime": "2018-03-16T15:19:47+09:00",
					"description": "[2018年03月16日]\\n  掲載"
				}
			]
		},
		"datefirstpublished": "2018-03-16T15:19:47+09:00",
		"datelastupdated": "2018-03-16T15:19:47+09:00",
		"datepublic": "2018-02-07T00:00:00+09:00"
	}
}
